### PR TITLE
Moved js code into .js file

### PR DESCRIPTION
**Pull request by italomaia**
It is now much easier to change the default behavior of the rendered js file.

### CODE REVIEW

**Comment by italomaia** [comment]
So?

File: flask_jsglue.py [hunk 1]
```diff
@@ -40,88 +43,11 @@ def context_processor():
...
-                }
-            }
-
-            throw {name: 'BuildError', message: "Couldn't find the matching endpoint."};
-        }
-    };
-});""" % (JSGLUE_NAMESPACE, json.dumps(rules))
+        # .js files are not autoescaped in flask
+        return render_template(
+            'jsglue/render.js',
```
Comment (stewartpark) [inline flask_jsglue.py]: nit: just a minor request, can we rename the filename to be `jsglue/js_bridge.js`? I think `render.js` isn't reflecting its functionality, it describes the action being done to the entity. 😁

**Review by stewartpark — APPROVED** [review]
Looks good, let's get the minor style fixed and merge it! Thanks @italomaia :)

**Comment by italomaia** [comment]
Seems fair. Done @stewartpark

**Pull request merged**